idaes/core/util/model_statistics.py

```diff
@@ -1270,7 +1282,7 @@ def degrees_of_freedom(block):
             number_activated_equalities(block))
 
 
-def large_residuals_set(block, tol=1e-5):
+def large_residuals_set(block, tol=1e-5, return_residual_values=False):
```

Comment:
The `return_residual_values` argument is not documented. Also, I think it's critical to document that the return type for the function will change depending on the value provided to that argument

Reply:
Great point.